qiskit_experiments/framework/composite/composite_analysis.py

```diff
@@ -94,7 +93,8 @@ def _run_analysis(self, experiment_data: ExperimentData, **options):
             sub_exp_data.add_data(sub_data)
 
             # Add component job metadata
-            sub_exp_data.metadata["job_metadata"] = [component_metadata[i]]
+            for key, val in component_metadata[i].items():
+                sub_exp_data.metadata[key] = val
```

Comment:
```suggestion
            sub_exp_data.metadata.update(component_metadata[i])
```
is this much simpler?

Reply:
Looks good